tests/ignite/handlers/test_state_param_scheduler.py

```diff
@@ -38,7 +38,7 @@
     },
 )
 
-if Version(torch.__version__) <= Version("1.7.0"):
+if Version(torch.__version__) < Version("1.9.0"):
```

Comment:
`torch.testing.assert_close` is introduced from `torch==1.9.0`

https://github.com/pytorch/pytorch/issues/61844